doc/make.bat

```diff
@@ -10,9 +10,11 @@ if "%SPHINXBUILD%" == "" (
 set SOURCEDIR=.
 set BUILDDIR=build
 set SPHINXPROJ=matplotlib
-set SPHINXOPTS=-W
-set O=
 
+if "%SPHINXOPTS%" == "" (
```

Comment:
Go for `if not defined SPHINXOPTS` https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/if.

That also makes it possible to use `set SPHINXOPTS=& make html` for setting an empty variable, i.e. we do not need the "set to anything but nothing" workaround.